src/onnx/parse_celu.cpp

```diff
@@ -0,0 +1,53 @@
+#include <migraphx/onnx/op_parser.hpp>
+#include <migraphx/onnx/checks.hpp>
+#include <migraphx/ranges.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/make_op.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace onnx {
+
+struct parse_celu : op_parser<parse_celu>
+{
+    std::vector<op_desc> operators() const { return {{"Celu"}}; }
+
+    instruction_ref parse(const op_desc&,
+                          const onnx_parser&,
+                          const onnx_parser::node_info& info,
+                          std::vector<instruction_ref> args) const
+    {
+        float alpha = 1.0;
+        if(contains(info.attributes, "alpha"))
+        {
+            alpha = info.attributes.at("alpha").f();
```

Comment:
We should check that alpha != 0. 
The white paper only discusses cases where alpha > 0, but I think alpha < 0 is still technically valid too.

Reply:
Yeah, alpha < 0 would be a pretty bad rectifier function as it quickly goes to large negative numbers.

Reply:
Checking for alpha != 0 doesn't pass CI because of comparing floating point numbers with == or !=. Could use an epsilon, but that doesn't really make much sense either. Maybe a check for alpha > 0 is the most reasonable?

Reply:
You should be able to just use `float_equal(alpha, 0.0f)`. I think float_equal.hpp is already indirectly included by op_parser.hpp